config.sample.yaml

```diff
@@ -50,6 +50,15 @@ ircService:
       # Whether or not to send connection/error notices to real Matrix users. Default: true.
       sendConnectionMessages: true
 
+      # Whether parts due to net-splits are debounced for debouncePartsMs, to allow
+      # time for the net-split to resolve itself. A net-split is detected as being
+      # a QUIT frequency higher than 5Hz. Default: false.
```

Comment:
This should be customisable rather than hard-coded to 5. This is why I mentioned about breaking this out into a separate config section like:
```yaml
debounce:
  enabled: true
  quitsPerSecond: 5    # If this is met or exceeded...
  waitTimeMs: 3600000  # Wait this amount of time for them to come back before parting them from Matrix
```